packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -68,6 +78,28 @@ export const firstComplementaryId = (active: Location['active']): string | undef
       : active.complementary
     : undefined;
 
+type WallpaperSlots = {
+  backgroundImage?: {
+    light?: string;
+    dark?: string;
+  };
+};
+
+const Wallpaper: FC<WallpaperSlots> = ({ backgroundImage: { dark, light } = {} }) => {
+  const { themeMode } = useThemeContext();
+  const backgroundImage = themeMode === 'dark' ? dark : light;
+  if (!backgroundImage) {
+    return <div className='absolute inset-0 z-0 bg-neutral-50 dark:bg-neutral-950' />;
+  }
+
+  return (
+    <div
+      className='absolute inset-0 z-0 bg-cover bg-no-repeat'
+      style={{ backgroundImage: `url(${backgroundImage})` }}
+    />
+  );
+};
```

Comment:
This should not be a component or add an element to the DOM, there are already several elements that can apply a `background-image` in an equivalent way.